views/common/detail_tabs/document.py

```diff
@@ -0,0 +1,583 @@
+                CONTEXT_GENERAL: ADMISSION_EMAIL_REQUEST_FAC_DOCUMENTS_GENERAL,
+                CONTEXT_DOCTORATE: ADMISSION_EMAIL_REQUEST_FAC_DOCUMENTS_DOCTORATE,
+                CONTEXT_CONTINUING: ADMISSION_EMAIL_REQUEST_FAC_DOCUMENTS_CONTINUING,
+            }
+            if ProgramManager.belong_to(self.request.user.person)
+            else {
+                CONTEXT_GENERAL: ADMISSION_EMAIL_REQUEST_SIC_DOCUMENTS_GENERAL,
+                CONTEXT_DOCTORATE: ADMISSION_EMAIL_REQUEST_SIC_DOCUMENTS_DOCTORATE,
+                CONTEXT_CONTINUING: ADMISSION_EMAIL_REQUEST_SIC_DOCUMENTS_CONTINUING,
+            }
+        )[self.current_context]
+
+        mail_template: MailTemplate = MailTemplate.objects.get_mail_template(
+            email_template_identifier,
+            current_language,
+        )
+
+        formation = self.proposition.doctorat if self.is_doctorate else self.proposition.formation
+
+        management_entity = (
+            EntityVersion.objects.filter(
+                acronym=formation.sigle_entite_gestion,
+                start_date__lte=current_date,
+            )
+            .exclude(end_date__lte=current_date)
+            .values('title')
+            .first()
+        )
+
+        tokens = {
+            'admission_reference': self.proposition.reference,
+            'training_campus': formation.campus,
+            'training_title': getattr(
+                self.admission.training,
+                'title' if current_language == settings.LANGUAGE_CODE_FR else 'title_english',
+            ),
+            'training_acronym': formation.sigle,
+            'training_year': format_academic_year(self.proposition.annee_calculee),
+            'admission_link_front': settings.ADMISSION_FRONTEND_LINK.format(
+                context=self.current_context,
+                uuid=self.proposition.uuid,
+            ),
+            'request_deadline': f'<span id="request_deadline">_</span>',  # Will be updated through JS
+            'management_entity_name': management_entity.get('title') if management_entity else '',
+            'management_entity_acronym': formation.sigle_entite_gestion,
+            'requested_documents': '<ul id="requested-documents-email-list"></ul>',  # Will be updated through JS
+        }
+
+        return mail_template.render_subject(tokens=tokens), mail_template.body_as_html(tokens=tokens)
+
+    def form_valid(self, form):
+        message_bus_instance.invoke(
+            ReclamerDocumentsAuCandidatParFACCommand(
+                uuid_proposition=self.admission_uuid,
+                identifiants_emplacements=form.cleaned_data['documents'],
+                auteur=self.request.user.username,
+                a_echeance_le=form.cleaned_data['deadline'],
+                objet_message=form.cleaned_data['message_object'],
+                corps_message=form.cleaned_data['message_content'],
+            )
+            if self.is_fac
+            else ReclamerDocumentsAuCandidatParSICCommand(
```

Comment:
je sais pas si c'est plus lisible

```suggestion
        cmd = ReclamerDocumentsAuCandidatParFACCommand if self.is_fac else ReclamerDocumentsAuCandidatParSICCommand
        cmd(...)
```